hack/build/docker/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora-minimal:30
+FROM registry.fedoraproject.org/fedora-minimal:29
```

Comment:
Can we find a good place to comment why we are stuck on 29 so that we will remember later and can know when to upgarade to 31?

Reply:
We are adding a functional test that imports the image that is giving us trouble. During the course of normal development we will attempt to upgrade the images, and if grab a version that is broken it should fail that test.